CONTRIBUTING.md

```diff
@@ -11,7 +11,8 @@ low-risk/isolated tasks:
 - Fix bugs found by [Coverity](#coverity).
 - [Merge a Vim patch] (requires strong familiarity with Vim)
   - NOTE: read the above link before sending improvements to "runtime files" (anything in `runtime/`).
-    - Vimscript and documentation files are (mostly) maintained by [Vim](https://github.com/vim/vim), not Nvim.
+    - Vimscript and documentation files are (mostly) maintained by [Vim], not Nvim.
+    - Nvim's filetype detection behavior should match Vim, so changes to filetype detection should be submitted to [Vim] first.
```

Comment:
should probably link to the relevant code file, else this item isn't doing much since anyone who didn't get it from the preceding item isn't going to get it from this item.
```suggestion
    - Nvim's [filetype detection](https://github.com/neovim/neovim/blob/master/runtime/lua/vim/filetype/detect.lua) behavior should match Vim, so changes to filetype detection should be submitted to [Vim] first.
```

Reply:
the entry point is `vim/filetype.lua`, though